testsuite/replay-verify/README.md

```diff
@@ -0,0 +1,12 @@
+context: https://www.notion.so/aptoslabs/WIP-Replay-Verify-Service-Design-1128b846eb728029bc51fcafb3de6fca?pvs=4
```

Comment:
I dont think this is externally visible? Can you include the tl;dr here or in a publicly accessible file?